tools/eslint/plugins/botbuilder.js

```
@@ -0,0 +1,41 @@
+
+module.exports = plugin = {
+    meta: {
+        name: 'eslint-plugin-botbuilder',
+        version: '1.0.0',
+    },
+    processors: {
+        disableLine: {
+            postprocess(messages) {
+                return []
+                    .concat(...messages)
+                    .filter((e) => !_config.disableLine[e.ruleId]?.some((pattern) => pattern.test(e.message)) ?? true);
```

ESLint

[error] Unexpected constant nullishness on the left-hand side of a `??` expression.

(no-constant-binary-expression)